app.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+require('env2')('./config.env');
+const reportRoutes = require('./routes/reportRoutes');
+
+mongoose.connect(process.env.MONGO_URL);
+
+const app = express();
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+
+app.get('/', (req, res) => {
+  res.send('Hello world');
+});
+
+app.use('/api', reportRoutes);
+
+const port = 5000 || process.env.PORT;
+app.listen(port, () => {
+  console.log(`App listening on http://localhost:${port}`);
```

Comment:
Can we add a URL variable instead of hardcoding localhost? (Im just thinking of when we will deploy... I know its far ahead but it's such a small detail, maybe we can do it now?) I know its just a tiny detail...

Reply:
Done @MynahMarie